### ISSUE TITLE
iPadOS 13 Checkboxes won't select/deselect with VO

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/gravity-force-lab-basics#176]
For https://github.com/phetsims/QA/issues/429
I was checking VO on iPadOS Beta since it will be coming out soon.  I was unable to check and uncheck boxes (Force Values, Distance, and Constant Size).  When attempting to double tap to do so, it would instead seem to activate the pink keyboard focus box and both it and the iPad's focus could be seen over the box.  Repeated attempts might cause it to read the box's description again, but it would not check or uncheck the box.  It seemed to work as expected in iOS 12.

**Comment by zepumph** [comment]
Assigning @jessegreenberg for assistance. Do you have a way to test this or reproduce it? I think we also may want to move this issue out of GFLB depending on its severity.

**Comment by jessegreenberg** [comment]
I confirmed it is working as expected in iOS 12. Ill see if I can get iOS 13 to test.

**Comment by jessegreenberg** [comment]
I have an iPad Air 2, https://www.apple.com/ipados/ says that will be compatible. iPadOS comes out tomorrow.

**Comment by zepumph** [comment]
Good luck!

**Comment by terracoda** [comment]
Is the iOS on iPad different from iOS on the iPhone?
I can check and uncheck checkboxes on iPhone 7 with iOS 12.4.1
Unfortunately, I haven't upgraded yet, so not sure how useful my test is.

**Comment by KatieWoe** [comment]
It is as of today's apple update.

**Comment by KatieWoe** [comment]
Not sure what it looks like on iOS 13 on iPhone.

**Comment by jessegreenberg** [comment]
OK, I confirmed that the checkbox cannot be checked in iPadOS13 while the cursor is on the checkbox. However, if the cursor is on the checkbox label, the checkbox can be checked correctly. Ill see if this is true for native HTML

**Comment by jessegreenberg** [comment]
https://jsfiddle.net/Lbnu98xq/ works OK.

It looks like the HTML checkboxes are not sized to overlap the entire Node with label:
![image](https://user-images.githubusercontent.com/6396244/65801139-603bc180-e146-11e9-946a-d78ed7636eab.png)

In chrome, the box takes up the entire text like this
![image](https://user-images.githubusercontent.com/6396244/65801179-7a759f80-e146-11e9-9b73-5a9c30e3900f.png)

Guessing width isn't being set correctly. Moving to scenery because I think this involves styling of PDOM elements.

**Comment by jessegreenberg** [comment]
The orange shows the dimensions of the HTML checkbox in Safari:
![image](https://user-images.githubusercontent.com/6396244/65802561-076e2800-e14a-11e9-94e6-7df6f302de7a.png)

But the interactive box isn't centered in this width.

**Comment by jessegreenberg** [comment]
In a separate HTML context, I saw that checkboxes cannot be clicked with `pointer-events: none` but the label could still be clicked. For this reason I thought that maybe `pointer-events: none` was preventing the checkbox from being clicked. But I confirmed that we are still receiving `touch` events on the checkbox in this case. So scenery should be able to receive. Maybe the pointer event isn't at the correct spot?

**Comment by terracoda** [comment]
In comment https://github.com/phetsims/scenery/issues/1001#issuecomment-536800772, @jessegreenberg has a solution, though it may be "fragile".

@zepumph, do we need to do another RC to test this out?

**Comment by zepumph** [comment]
@terracoda we will definitely want another RC for this. The commits over in https://github.com/phetsims/scenery/issues/1001 will be cherry picked to the branch. Then we will have QA look at this.

**Comment by KatieWoe** [comment]
Issue seems solved.  Checked by @loganbraywork
